main.nf

```diff
@@ -3,119 +3,153 @@
 // Meta-workflow that controls execution of steps in cell type 
 // prediction tools evaluation framework
 
-// extract reference and query data
-if(params.data_download.run == "True"){
-    query_data = params.data_download.query_output_dir
-    query_n_clust = params.data_download.query_num_clust.toString()
-    query_markers = "marker_genes_" + query_n_clust + ".tsv"
+// download data
+if(params.data_import.run == "True"){
+
+	// initialise required channels 
+	DATASET_IDS = Channel.create()
+	NUM_CLUST = Channel.create()
+	BARCODE_COLUMN = Channel.create()
```

Comment:
Seems like you're not using the above channels (except `NUM_CLUST`) anywhere else in the script. Is that supposed to be so? If yes, I think we can get rid of them.

Reply:
Right, none of them are used later on (as NUM_CLUST is empty, it's N_CLUST the one carrying the value). I'm erasing them

Reply:
Actually, I copied the chunk from [here](https://github.com/ebi-gene-expression-group/cell-types-train-control-workflow/blob/bf0b9cb5b8ce8615768ba83a85deea73e9c1c7ff/main.nf#L4), but what are these channels for?